src/server/api/routes/users-router.js

```diff
@@ -0,0 +1,182 @@
+const express = require('express');
+
+const router = express.Router({
+  mergeParams: true,
+});
+
+// controllers
+const usersController = require('../controllers/users-controller');
+
+/**
+ * @swagger
+ * /users:
+ *  get:
+ *    summary: Get all users
+ *    description:
+ *      Will return all users.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  usersController
+    .getUsers()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /users/{ID}:
+ *  get:
+ *    summary: Get module by ID
+ *    description:
+ *      Will return single module with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the module to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/:id', (req, res, next) => {
+  usersController
+    .getUserById(req.params.id)
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /users:
+ *  post:
+ *    summary: Create a module
+ *    description:
+ *      Will create a module.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: module
+ *        description: The module to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - title
+ *            - startDate
+ *            - endDate
+ *             - classId
+ *          properties:
+ *            title:
+ *              type: string
+ *            startDate:
+ *              type: string
+ *              format: date-time
+ *            endDate:
+ *              type: string
+ *              format: date-time
+ *            classId:
+ *              type: string
+ *    responses:
+ *      201:
+ *        description: Module created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
```

Comment:
Why is this here? Your ticket mentions only implementing GET and this method `usersController.createUser` doesn't even exist! And the comment mentions the word module. Please remove.